kernel/syscall.c

```diff
@@ -2087,11 +2088,23 @@ long myst_syscall_execve(
     long ret = 0;
     const char** argv = NULL;
     myst_thread_t* current_thread = myst_thread_self();
+    myst_path_t* resolved_path = NULL;
```

Comment:
`myst_path_t` is a struct with a char array. You can just declare the variable instead of a pointer.

Reply:
It would breach stack size limit if we do that

`error: stack usage might be 4272 bytes [-Werror=stack-usage=]`

Reply:
Not sure if it's configurable, can you check? 4272 bytes does not seem to be large to me though.

Reply:
The error could be ignored with `#pragma GCC diagnostic ignored "-Wstack-usage="`, but I am not sure if its a good idea, and couldn't find the place to configure the threshold... Do you have more info on this issue?

Reply:
ok, I see we set the limit to 512: https://github.com/deislabs/mystikos/blob/6cdeb80aef0faf7e807b953c44d241f3543a5676/kernel/Makefile#L42

Let's keep the current implementation.